### PR TITLE
Update URL of "tdslite" 

### PR DESCRIPTION
**Pull request by mustafakemalgilor**
"tdslite" has been moved to the org with the same name

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @mustafakemalgilor. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/12/06 17:04:41 Loaded 6428 libraries from DB
Changing URL of library tdslite from https://github.com/mustafakemalgilor/tdslite.git to https://github.com/tdslite/tdslite.git
Success!
```

**Pull request merged**
